src/proxyless/resource-injector.ts

```diff
@@ -20,6 +20,7 @@ import debug from 'debug';
 import { StatusCodes } from 'http-status-codes';
```

Comment:
Why didn't you move calls of other `client` methods to `utils`?

Reply:
The `redirect` and `navigateTo` methods are common utility methods. They are used in a few places.
Note: the `redirect` method is used in one place but it is analog of `navigateTo` method.
If you know other utility methods, please write them.